Add 10.7 release notes (#32)
mondoohq/docs: releases/2024-03-12-mondoo-10.7-is-out.md

---
slug: mondoo-10.7-is-out/
title: Mondoo 10.7 is out!
author: Tim Smith
author_title: Mondoo Core Team
author_url: https://github.com/tas50
tags: [release, mondoo]
---

## 🥳 Mondoo 10.7 is out! This release includes vendor advisory links, improved CLI scanning, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🧹 IMPROVEMENTS

> ### Show numeric asset scores on the CLI

Reviewer: ```suggestion
### Show numeric asset scores in the CLI
```

> Understand your precise scores on the CLI with new numeric score values in addition to A-F scores

Reviewer: ```suggestion
Understand your precise scores in the cnspec CLI with new numeric score values in addition to A-F scores.
```

![Numeric scores](/img/releases/2024-03-12-mondoo-10.7-is-out/numeric.png)

### Add specific vendor advisory sources

> Jump right to the source with new direct links to vendor advisories in software advisory pages.

Reviewer: ```suggestion
Jump right to the source with new direct links to vendor advisories on software advisory pages.
```

![Advisory links](/img/releases/2024-03-12-mondoo-10.7-is-out/vendor_links.png)

> ### Improved AWS Integration troubleshooting

Reviewer: ```suggestion
### Improved AWS integration troubleshooting
```

> Failures happen, so let's get to the root cause faster with new troubleshooting options for AWS integrations. The ... menu in the AWS integrations pages now includes options to force an update of the Lambda code powering the integration along with new options for sending diagnostics logs directly to Mondoo.

Reviewer: ```suggestion
Failures happen, so let's get to the root cause faster with new troubleshooting options for AWS integrations. The ... menu in the AWS integrations pages now includes new options that:
- Force an update of the Lambda code powering the integration
- Send diagnostics logs directly to Mondoo
```

![Diagnostics information](/img/releases/2024-03-12-mondoo-10.7-is-out/aws_diagnostics.png)

## 🐛 BUG FIXES AND UPDATES

> - Fix failures to detect vulnerable versions of system-wide Visual Studio Code installs on Windows.

Reviewer: ```suggestion
- Fix failures to detect vulnerable versions of system-wide Visual Studio Code installations on Windows.
```
Am I being too formal?

Author: I'm not attached to it the other way

- Fix incorrect pluralization on the assets page.

> - Fix incorrect source links for Debian, Chrome, and Firefox vulnerabilities and Advisories.

Reviewer: ```suggestion
- Fix incorrect source links for Debian, Chrome, and Firefox vulnerabilities and advisories.
```

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🧹 IMPROVEMENTS

### Show numeric asset scores in the CLI

Understand your precise scores in the cnspec CLI with new numeric score values in addition to A-F scores.

![Numeric scores](/img/releases/2024-03-12-mondoo-10.7-is-out/numeric.png)

### Add specific vendor advisory sources

Jump right to the source with new direct links to vendor advisories on software advisory pages.

![Advisory links](/img/releases/2024-03-12-mondoo-10.7-is-out/vendor_links.png)

### Improved AWS integration troubleshooting

Failures happen, so let's get to the root cause faster with new troubleshooting options for AWS integrations. The ... menu in the AWS integrations pages now includes options to force an update of the Lambda code powering the integration along with new options for sending diagnostics logs directly to Mondoo.

![Diagnostics information](/img/releases/2024-03-12-mondoo-10.7-is-out/aws_diagnostics.png)

## Kubernetes scanning performance improvements

> Introduce a new mechanism to reduce the number of calls made during asset discovery. This is helpful when scanning e.g. larger Kubernetes clusters, and allows cnquery/cnspec to incrementally scan every asset one by one without having to scan all of them initially. This performance improvement not only cuts drastically the execution time, it also eliminates the need for reading container images twice from the system, cutting down on I/O load.

Reviewer: ```suggestion
We introduced a new mechanism to reduce the number of calls made during asset discovery. This is especially helpful when scanning larger Kubernetes clusters. It lets cnquery and cnspec incrementally scan every asset one by one without having to scan all of them initially. This performance improvement not only drastically cuts the execution time, it also eliminates the need for reading container images twice from the system, cutting down on I/O load.
```

This improvement is automatically enabled for new workloads. We currently support it for container images and plan to extend it to other workloads with costly discovery steps in the future.

## 🐛 BUG FIXES AND UPDATES

- Fix failures to detect vulnerable versions of system-wide Visual Studio Code installations on Windows.
- Fix incorrect pluralization on the assets page.
- Fix incorrect source links for Debian, Chrome, and Firefox vulnerabilities and advisories.
- Fix detection of some newer VMware advisories.
- Fix macOS systems displaying a low vulnerability score but no CVEs or advisories.
- Add missing available package data when scanning for vulnerabilities on the command line.
- Fix failures scanning systems with the command line `--incognito` flag.
- Add missing first-found data to the asset software tab.
- Respect the `--output` flag when running `cnspec vuln`.
- Improve the disk/memory usage of container image scans on large Kubernetes clusters.
- Fix duplicate AWS instance scans.
- Add support for VMware vSphere/ESXi 8.0U2b vulnerability scanning.
- Don't show the service accounts button when a Kubernetes integration is still pending.

> - Show unknown instead of 0.0 when a CVSS score has not been published.

Reviewer: ```suggestion
- Show "unknown" instead of "0.0" when a CVSS score has not been published.
```
